xl-v1-multipart.go

```diff
@@ -716,6 +710,7 @@ func (xl xlObjects) CompleteMultipartUpload(bucket string, object string, upload
 	// Save the final object size and modtime.
 	xlMeta.Stat.Size = objectSize
 	xlMeta.Stat.ModTime = time.Now().UTC()
+	xlMeta.Stat.UUID = getUUID()
```

Comment:
do we need a new UUID during completeMultipartUpload? Can't we retain the UUID assigned during putObjectPart request for the same uploadID?